### DISCUSSION TITLE
firewall questions

### DISCUSSION THREAD

**Discussion by vector1one** [Security-Onion-Solutions/securityonion#1544] [Unsupported Versions]
I'm getting issues with SO2 and my firewall...i have a pfsense with NAT/port forwarding
Im trying to send logs over 5044 but its not getting to SO2
I have used the so-allow and added in my firewall. If I send logs to the SO2 from with in the firewall, no issue, only from outside. All my other services work with the same NAT setting, its just the SO2...anything I need to manage in SO2 to enable:
Workstation(5044) > firewall{NAT/PORTFWD-5044] > SO2(5044)

cheers

**Comment by weslambert** [comment]
@vector1one are you still experiencing issues with this?
